contracts/actions/post/collect/ISimpleCollectAction.sol

```diff
@@ -12,9 +12,18 @@ import {RecipientData} from "contracts/core/types/Types.sol";
  * @param collectLimit The maximum number of collects for this publication. 0 for no limit.
  * @param token The token associated with this publication.
  * @param currentCollects The current number of collects for this publication.
- * @param recipient Recipient of collect fees.
+ * @param recipients Recipients of collect amount.
  * @param endTimestamp The end timestamp after which collecting is impossible. 0 for no expiry.
+ * @param referralFeeBps The referral fee in basis points.
+ * @param followerOnlyGraph  The graph that holds the follow relations that restrict who can collect this post.
  * @param collectionAddress The address of the collectible ERC721 contract.
+ * @param isImmutable If true, it means that:
+ *          - The Post URI is snapshotted at configuration time and cannot be changed later.
+ *          - Collected posts' NFTs remain permanently available.
+ *          - What you see is what you get; editing the Post URI or deleting the post will disable collection.
+ *         Note: This immutability is only guaranteed if the URI is hosted on immutable storage. Mutability inherent
+ *         to the chosen storage technology exceeds the on-chain verification capabilities.
+ * @param isDisabled Whether the collect action is disabled or not.
  */
 struct CollectActionData {
```

Comment:
if this is emitted in any event anywhere we cant change this naming if not all is ok

Reply:
No, it's not emitted. Still if emitted the important things are the types, not the names of the struct fields, so it would have been ok. But not emitted.